eng/pipelines/runtimelab.yml

```diff
@@ -51,3 +51,33 @@ jobs:
 # Checkout repository
 #
 - template: /eng/pipelines/common/checkout-job.yml
+
+#
+# Build with Debug config and Release runtimeConfiguration
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: release
+    platforms:
+    - Linux_x64
+    - Windows_NT_x64
+    jobParameters:
+      testGroup: innerloop
+      nameSuffix: Runtime_Debug
+      buildArgs: -c debug -runtimeConfiguration release
```

Comment:
These two will build mono, do we want mono in this experiment? 

The default subsets are: clr+mono+libs+installer: https://github.com/dotnet/runtime/blob/34fd21edebf010cfc9a9d685e4ad22d06f916fc1/eng/Subsets.props#L40

Reply:
oh good catch, missed that. No we want coreclr for now only, I'll fix that.